lib/train/transports/clients/azure/graph_rbac.rb

```diff
@@ -1,28 +1,32 @@
 # encoding: utf-8
+# frozen_string_literal: true
+
 require 'azure_graph_rbac'
 
 # Wrapper class for ::Azure::GraphRbac::Profiles::Latest::Client allowing custom configuration,
 # for example, defining additional settings for the ::MsRestAzure::ApplicationTokenProvider.
 class GraphRbac
-  AUTH_ENDPOINT = MsRestAzure::AzureEnvironments::AzureCloud.active_directory_endpoint_url
-  API_ENDPOINT = MsRestAzure::AzureEnvironments::AzureCloud.active_directory_graph_resource_id
+  def initialize
+    @auth_endpoint = MsRestAzure::AzureEnvironments::AzureCloud.active_directory_endpoint_url
+    @api_endpoint = MsRestAzure::AzureEnvironments::AzureCloud.active_directory_graph_resource_id
+  end
```

Comment:
I can't see why you want to make this class stateful. What's the benefit?

Reply:
You're right - it was unnecesary, i've changed it back!